frontend/model/state.js

```diff
@@ -455,7 +455,7 @@ const getters = {
   currentWelcomeInvite (state, getters) {
     const invites = getters.currentGroupState.invites
     const inviteId = Object.keys(invites).find(invite => invites[invite].creatorID === INVITE_INITIAL_CREATOR)
-    const expires = getters.currentGroupState._vm.authorizedKeys[inviteId].meta.expires
+    const expires = getters.currentGroupState._vm.invites[inviteId].expires
```

Comment:
@corrideat Can you review this?

Reply:
Let me know if this should be reverted.
FYI, I noticed there is many other places where a group's invites are used this way.

eg1.
https://github.com/okTurtles/group-income/blob/9fd5e1643cf0396ad0dfb58b6d453ddb941a892c/frontend/model/contracts/shared/getters/group.js#L133-L136

eg2.
https://github.com/okTurtles/group-income/blob/9fd5e1643cf0396ad0dfb58b6d453ddb941a892c/frontend/views/containers/group-settings/InvitationLinkModal.vue#L48-L54

eg3.
https://github.com/okTurtles/group-income/blob/9fd5e1643cf0396ad0dfb58b6d453ddb941a892c/frontend/views/containers/proposals/ProposalItem.vue#L308-L313

Reply:
I think this change should be fine, provided that the key name starts with `#inviteKey`

Reply:
To expand on this, some of the `meta` attributes are copied to the `.invites` field. I think it makes more sense to check under the dedicated `.invites` property, unless there's some reason not to. For example, if one wants access to the original attributes as they key was created, one would use `_vm.authorizedKeys`.